core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -352,7 +352,7 @@ class SparkContext(config: SparkConf) extends Logging {
 
   // Thread Local variable that can be used by users to pass information down the stack
   protected[spark] val localProperties = new InheritableThreadLocal[Properties] {
-    override protected def childValue(parent: Properties): Properties = {
+    override def childValue(parent: Properties): Properties = {
```

Comment:
[ERROR] [Error] /spark/core/src/main/scala/org/apache/spark/SparkContext.scala:355: weaker access privileges in overriding
private[package lang] def childValue(x$1: java.util.Properties): java.util.Properties (defined in class ThreadLocal)
  override should at least be private[lang]